assets/schema_input.json

```diff
@@ -65,43 +65,43 @@
                 "format": "file-path",
                 "exists": true,
                 "pattern": "^\\S+\\.(bam|fastq.gz|fq.gz)$",
-                "errorMessage": "Long read files must be in BAM or GZIPed FASTQ format and cannot contain spaces."
+                "errorMessage": "Path to long read files must exist, end in '.bam' or '.fastq.gz' or '.fq.gz', and cannot contain spaces."
             },
             "reads1": {
                 "type": "string",
                 "format": "file-path",
                 "exists": true,
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
-                "errorMessage": "Path to first short reads file (_R1 in a pair) must be provided and cannot contain spaces"
+                "errorMessage": "Path to first short reads file (_R1 in a pair) must exist, end in '.fastq.gz' or '.fq.gz', and cannot contain spaces."
             },
             "reads2": {
                 "type": "string",
                 "format": "file-path",
                 "exists": true,
                 "pattern": "^\\S+\\.f(ast)?q\\.gz$",
-                "errorMessage": "Path to paired short reads file (_R2) cannot contain spaces"
+                "errorMessage": "Path to paired short reads file (_R2 in a pair) must exist, end in '.fastq.gz' or '.fq.gz', and cannot contain spaces."
             },
             "aligned_reads": {
                 "type": "string",
                 "format": "file-path",
                 "exists": true,
                 "pattern": "\\S+\\.(bam|cram)$",
-                "errorMessage": "Aligned reads BAM or CRAM, cannot contain spaces and must have extension '.bam' or '.cram'"
+                "errorMessage": "Path to aligned reads must exist, end in '.bam' or '.cram', and cannot contain spaces."
             },
             "fastp_json": {
                 "type": "string",
                 "format": "file-path",
                 "exists": true,
                 "pattern": "\\S+\\.(json)$",
-                "errorMessage": "fastp JSON report from reads in alignment",
+                "errorMessage": "Path to fastp JSON report must exist, end in '.json', and cannot contain spaces.",
```

Comment:
This now can also accept  FASTPLONG results, would be nice to add to the description.